python/mlx/data/datasets/mnist.py

```diff
@@ -69,3 +66,32 @@ def download():
     pkl_file = (root / "train.pkl") if train else (root / "test.pkl")
     with pkl_file.open("rb") as f:
         return dx.buffer_from_vector(pickle.load(f))
+
+
+def load_mnist(root=None, train=True):
+    """Load a buffer with the MNIST dataset.
+
+    If the data doesn't exist download it and save it for the next time.
+
+    Args:
+        root (Path or str, optional): The directory to load/save the data. If
+            none is given the ``~/.cache/mlx.data/mnist`` is used.
+        train (bool): Load the training or test set.
+    """
+
+    return _load_mnist_wrapper(root, train, "mnist")
+
+
+def load_fashion_mnist(root=None, train=True):
+
```

Comment:
Oh missed that initially. Could you remove the empty line and make sure you run pre-commit ? Thanks.

Reply:
Fixed and `pre-commit run --all-files` all passed now.